server/config/config.json.example

```diff
@@ -1,4 +1,5 @@
 {
+  "port":3000,
```

Comment:
The port should be inside development, staging, etc.

Reply:
Done